Core/build.gradle

```diff
@@ -8,6 +8,13 @@ java {
     targetCompatibility = JavaVersion.VERSION_1_7
 }
 
+compileKotlin {
```

Comment:
because 'kotlin.Result' cannot be used as a return type. so we need use this flag. [stackoverflow topic](https://stackoverflow.com/questions/52631827/why-cant-kotlin-result-be-used-as-a-return-type)